docs/localization.md

```diff
@@ -21,6 +21,28 @@ let title = NSLocalizedString("Following %@", comment: "")
 
 Comments help give more context to translators.
 
+## Always Use Positional Placeholders
+
+Always include the positional index of parameters. For example, instead of just using `%@` as the placeholder, use `%1$@` instead. Positional placeholders allow translators to change the order of placeholders or repeat them if necessary.
```

Comment:
I think it could deserve a quick mention of "apply this rule even if there's only one placeholder", because it's common to think about this only when there are multiple.

```suggestion
Always include the positional index of parameters, even if you only have one placeholder in your string. For example, instead of using `%@` as the placeholder, use `%1$@` instead. Positional placeholders allow translators to change the order of placeholders or repeat them if necessary.
```

+ Add an example, with single placeholder, in the Do / Don't examples snippet below, in addition to the ones you already wrote.

Reply:
Thanks for the feedback. I thought it was obvious. 🤷 Fixed.

Reply:
I don't see the additional example with a single placeholder I suggested about though 😅  (But maybe I'm overthinking / overdocumenting it 😓 )

Reply:
Oh, I thought it was okay because there are already a lot of examples with single placeholders in the doc. So we might be adding too much information that readers would probably just skip anyway.

Reply:
That's a good point. Yeah, probably not worth adding too many examples then you're right 👍